scripts/minikube-minishift-setup-env.sh

```diff
@@ -7,10 +7,11 @@ shout() {
 }
 
 # Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
-mkdir bin artifacts
+mkdir -p go/bin artifacts
```

Comment:
we should make sure that the script can be executed from any directory.
If `go/bin` and `artifacts` are supposed to create in home it should use `$HOME`

Reply:
no, not in home. It will be same as not creating at all. We want unique GOPATH and GOBIn for each run

Reply:
I think it would be safer to have a different $HOME for each script running, as lots of tools are relying on this variable to set files.
If necessary, we could have a bunch of users available for the different scripts, or create these users on the fly.